exchange/order/limits/limits_types.go

```diff
@@ -69,4 +70,7 @@ type MinMaxLevel struct {
 	MarketStepIncrementSize float64
 	MaxTotalOrders          int64
 	MaxAlgoOrders           int64
+	Delisting               bool
```

Comment:
🥃 Debating if it's that useful to have this, versus `!DelistingTime.IsZero()`
You probably had the same thought.
```suggestion
	Delisting               bool
```

Reply:
Yeah it is a double up of fields, probably just remove it to be honest. Thanks.